### ISSUE TITLE
--d4 option unavailble in 0.3.4

### ISSUE THREAD

**Issue by aarthi-mohan** [brentp/mosdepth#206]
Hi Brent,

I installed mosdepth via conda, and seems like the `--d4` is unavailable in the above version. Here is the screenshot
```
(mosdepth)[am]$ mosdepth --version 
mosdepth 0.3.4
(mosdepth)[am]$ mosdepth -t 8 --d4  -f genomes/hg38.fa 213470-mosdepth_perbase_d4 213470.cram
Usage: mosdepth [options] <prefix> <BAM-or-CRAM>
error parsing arguments
```

Thanks,
Aarthi

**Comment by kenissur** [comment]
> Hi, are you using d4? It's hard to maintain support so I'd actually like to remove it.

Hi. I was using BigWig files elsewhere, d4 sounded like a good alternative (e.g. has a python API) but .bed.gz is not. It's also confusing that the --d4 option appears in the README but not in the actual tool in the same version. What are the plans for this?

**Comment by brentp** [comment]
Hi, --d4 is available in the latest release, v0.3.5. I suggest that you use that.
